### DISCUSSION TITLE
PPOCRLabel标注单元格的换行问题

### DISCUSSION THREAD

**Discussion by zacQin** [PaddlePaddle/PaddleOCR#12716] [Q&A]
![1](https://github.com/PaddlePaddle/PaddleOCR/assets/33820355/676ee51f-5efa-47fa-babf-ccb0e3843a0e)
![2](https://github.com/PaddlePaddle/PaddleOCR/assets/33820355/6f195d59-f26a-4f67-a84f-c97ab050c626)
表格单元格的内容有换行，根据2[表格标注](https://github.com/PaddlePaddle/PaddleOCR/blob/release/2.7/PPOCRLabel/README_ch.md#21-%E6%93%8D%E4%BD%9C%E6%AD%A5%E9%AA%A4)的第二条
2.更改标注结果：以表格中的单元格为单位添加标注框（即一个单元格内的文字都标记为一个框）
规则合并在一起
那么在PPOCRLabel的界面上应该怎么标注，用空格表示吗？如果用空格表示，那真正的空格应该怎么表示？

**Comment by SWHL** [comment]
因PPOCRLabel已经搬家了，后续有问题请移步[PPOCRLabel](https://github.com/PFCCLab/PPOCRLabel)提问哈！
此issue先行关闭了。
